buildSrc/src/main/groovy/io/vantiq/extsrc/assy/tasks/AssemblyResourceGeneration.groovy

```diff
@@ -523,14 +546,18 @@ class AssemblyResourceGeneration extends DefaultTask {
                 ((String) pDesc[PROPERTY_X_DESCRIPTORS])?.contains(DISPLAY_HIDE_VALUE)) {
                 // for things that are passwords or are marked to be masked in display, we will require a secret.
                 propDesc.type = 'Secret'
-                propDesc.description += ' (Please provide the name of the Vantiq Secret containing ' + 'this value.)'
+                propDesc.description += ' (Please provide the name of the Vantiq Secret containing this value.)'
             }
             if (pDesc.enum != null) {
-                propDesc.enum = pDesc.enum
```

Comment:
Re: the section below.  This speculates on the interface that may work.  Whether this survives until publication or not remains to be seen.  But it works as is -- if this interface is adopted, then we'll immediately take advantage of it.